### PR TITLE
Need to merge coding bat 2

### PR DESCRIPTION
**Pull request by srmoore5**
it is here
http://srmoore5.github.io/spring2014/2014/02/16/secondcodingbatsierra.html

Also added my nitrous test post

http://srmoore5.github.io/spring2014/2014/02/19/sierrapost.html

### CODE REVIEW

File: _posts/2014-02-16-2codingbatsierra.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+author: srmoore5
+title: Coding Bat 2
+date: 02/16/2014
```
Comment (gerbal) [inline _posts/2014-02-16-2codingbatsierra.md]: It looks like Jekyll doesn't recognize this date format. Either phrase it as year-month-day, day-month-year or leave it out outright and Jekyll will just get the date from the filename.

**Comment by eah13** [comment]
@srmoore5, your code blocks are messed up.  Take a second look at your page.  There's a missing empty line in front of some of your backticks.

I'd also love to see some more in-depth reflections next time.

**Comment by srmoore5** [comment]
Fixed

**Comment by gerbal** [comment]
The absence of a date on the date line of the header may be preventing the page from building correctly

File: _posts/2014-02-16-secondcodingbatsierra.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+author: srmoore5
+title: Coding Bat 2
+date:
```
Comment (eah13) [inline _posts/2014-02-16-secondcodingbatsierra.md]: @srmoore5 this is what grant was talking about.  Your preview is still broken- take a second look.

**Comment by eah13** [comment]
Nice work! Strange crystal cave.  

I'd love to hear more about your process & reflections next time or in class.

**Pull request merged**